clients/unshaded/src/main/java/tachyon/client/file/options/CreateFileOptions.java

```diff
@@ -178,9 +178,8 @@ public String toString() {
   /**
    * @return Thrift representation of the options
    */
-  // TODO(calvin): Rename CreateTOptions to CreateFileTOptions
-  public CreateTOptions toThrift() {
-    CreateTOptions options = new CreateTOptions();
+  public CreateFileTOptions toThrift() {
```

Comment:
Ideally I would like a thrift representation of all our options instead of sometimes extracting fields to fill method args. I've opened a ticket here: https://tachyon.atlassian.net/browse/TACHYON-1546